tests/run_selective.py

```
@@ -57,18 +65,18 @@ def main() -> None:
-		if (sys.stdout.isatty()):
-			print(f"{_bar}START{_bar}", file=sys.stdout)
+		logger = logging.getLogger(__module__)
+		logger.info(f"{_bar}GROUP{_bar}")  # skipcq PYL-W1203 - test code ok
+		logger.info(f"{args.group}:{args.category}")  # skipcq PYL-W1203 - test code ok
+		logger.info(f"{_bar}START{_bar}")  # skipcq PYL-W1203 - test code ok
 		suite = get_test_suite(args.group, args.category)
 		runner = unittest.TextTestRunner(verbosity=2)
 		result = runner.run(suite)
-		if (sys.stdout.isatty()):
-			print(f"{_bar} END {_bar}", file=sys.stdout)
+		logger.info(f"{_bar} END {_bar}")  # skipcq PYL-W1203 - test code ok
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)